plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/KnowledgeArtifactProcessor.java

```diff
@@ -2,15 +2,7 @@
 import static java.util.Comparator.comparing;
 
 import java.time.Instant;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.TreeSet;
-import java.util.UUID;
+import java.util.*;
```

Comment:
nice

Reply:
Fixed

Reply:
I wasn't asking for changes sorry, it did look cleaner 🙈